src/modules/editor/components/MenuBar.tsx

```diff
@@ -261,23 +270,32 @@ const AddExtensionDialog: React.FC<IAddItemProps> = ({ extensionInfo, state, dis
 
     try {
         const parsedProps = props ? JSON.parse(props) : undefined
+        const parsedMediaIds = mediaIds ? JSON.parse(mediaIds) : []
         return (
-            <Dialog open={!!extensionInfo} props={parsedProps} onSubmit={(e) => {
-                onClose()
-                if (e === null) {
-                    ;
-                }
-                else {
-                    insertNode(schema.node(schema.nodes.article_extension, {
-                        type,
-                        props: JSON.stringify(e)
-                    }))(state, dispatch)
-                }
-            }} />
+            <Dialog 
+                open={!!extensionInfo} 
+                props={parsedProps} 
+                allMedia={media}
+                mediaIds={parsedMediaIds} 
+                onMediumAdd={onMediumAdd} 
+                onSubmit={(e) => {
+                    onClose()
+                    if (e === null) {
+                        ;
```

Comment:
?

Reply:
Good catch, that's unneeded I think. Probably the result of copy paste.